zeeguu_api/api/dashboard.py

```diff
@@ -0,0 +1,365 @@
+from datetime import datetime
+
+import flask
+from flask import request
+
+from .utils.json_result import json_result
+from .utils.route_wrappers import cross_domain, with_session
+from . import api
+import zeeguu
+from zeeguu.model import User, Cohort, UserActivityData, Session, Language
+import sqlalchemy
+from flask import jsonify
+import json
+import random
+import datetime
+
+db = zeeguu.db
+
+
+def _has_permission_for_cohort(cohort_id):
+    '''
+        Checks to see if user requesting has permissions to view the cohort with id 'cohort_id'
+    '''
+    from zeeguu.model import TeacherCohortMap
+    links = TeacherCohortMap.query.filter_by(cohort_id=cohort_id).all()
+    for l in links:
```

Comment:
still don't like this for l in links.
i think later you use `for m in maps` which is slightly better.